sbin/install.config.in

```diff
@@ -0,0 +1,3 @@
+DATAROOTDIR="$datarootdir_POST"
+SYSCONFDIR="$sysconfdir_POST"
+LOCALSTATEDIR="$localstatedir_POST"
```

Comment:
not sure why we need several template configuration files, can't you use only one?